db/migrations/20220429093237-create-table-request.ts

```diff
@@ -0,0 +1,43 @@
+import {QueryInterface} from "sequelize";
+import {Sequelize_migration} from "../util/inteface";
+
+
+export const up = async (queryInterface: QueryInterface, Sequelize: Sequelize_migration) => {
+    await queryInterface.createTable('request', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      from: {
+        type: Sequelize.INTEGER
+      },
+      to: {
+        type: Sequelize.INTEGER
+      },
+      status: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        defaultValue: 'decline'
+      },
+      type: {
+        type: Sequelize.STRING,
```

Comment:
what does it represent? Perhaps worth creating an enum for this col? Same for the `status` col